e2e/test-runner/runner_test.go

```diff
@@ -1,10 +1,23 @@
 package test_runner
 
 import (
+	"gopkg.in/ini.v1"
 	"strings"
 	"testing"
+	"time"
 )
 
+func expectedUpgradePolicy() string {
+	if *distroName == "Ubuntu" {

```

Comment:
I'd recommend passing the pointer to the distroName as an argument, since `expectedUpgradePolicy()` is meant to be called in the middle of a higher-level function. We may change the global variable name later in the future, or the way it's set (causing it to even not be a global anymore).